packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -154,16 +168,23 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
   };
 
   private _onChange = (ev: React.FormEvent<HTMLElement>): void => {
-    const { disabled, onChange } = this.props;
-    const { isChecked } = this.state;
+    const { disabled, onChange, removeIndeterminate, indeterminate, defaultIndeterminate } = this.props;
```

Comment:
Not the right line for this, but you should add an `indeterminate: boolean` property to `ICheckbox`, and a corresponding getter in class (mirroring `public get checked`).

Reply:
Very good catch :clap: